config/opal_functions.m4

```diff
@@ -96,7 +96,7 @@ EOF
 
 OPAL_CONFIGURE_USER="${USER:-`whoami`}"
 OPAL_CONFIGURE_HOST="${HOSTNAME:-`(hostname || uname -n) 2> /dev/null | sed 1q`}"
-OPAL_CONFIGURE_DATE="`date`"
+OPAL_CONFIGURE_DATE="`date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}"`"
```

Comment:
Is there a reason to not use `config/getdate.sh` here?

Reply:
Would `$(top_srcdir)` be available here?

Reply:
It looks like `$(srcdir)` is available -- is that sufficient?